arex-schedule-web-api/src/main/java/com/arextest/schedule/planexecution/PlanExecutionMonitorImpl.java

```diff
@@ -40,9 +45,11 @@ public class PlanExecutionMonitorImpl implements PlanExecutionMonitor {
     @Value("${arex.schedule.bizLog.secondToSave}")
     private long LOG_TIME_GAP_TO_SAVE_CHECK_BY_SEC;
 
-    private final RedisCancelMonitor cancelMonitor = new RedisCancelMonitor();
     private final ProgressMonitor progressMonitor = new ProgressMonitor();
     private final BizLoggerMonitor bizLoggerMonitor = new BizLoggerMonitor();
+    private final RedisCancelMonitor redisCancelMonitor = new RedisCancelMonitor();
+    private static final ConcurrentHashMap<String, List<ReplayPlanStageInfo>> replayPlanStageListMap =
```

Comment:
Can be done by Map<String, int>, we only need the list size to detect modification